controllers/reset.controller.js

```diff
@@ -0,0 +1,40 @@
+const { resetService, findService } = require("../services/auth-user.service");
```

Comment:
reset.controller should be a part of auth.controller. Same for service and routes